gulpfile.js

```diff
@@ -0,0 +1,101 @@
+var _ = require('lodash');
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var ts = require('gulp-typescript');
+var tslint = require('gulp-tslint');
+var realTslint = require('tslint');
+var spawnSync = require('child_process').spawnSync;
+
+function build(src, dest) {
+  var tsProject = ts.createProject('configs/tsconfig-base.json', { declaration: true });
+  return gulp.src(src)
+    .pipe(tsProject())
+    .pipe(gulp.dest(dest));
+}
+
+gulp.task('build:capnp', function () {
+  return build('./packages/capnp-ts/src/**/*.ts', 'packages/capnp-ts/lib');
+});
+
+gulp.task('build:capnpc', function () {
+  // Stub, schema are not compiled yet
+  return build('./packages/capnpc-ts/src/**/*.ts', 'packages/capnpc-ts/lib');
+});
+
+gulp.task('build', ['build:capnp'/*, 'build:capnpc'*/]);
```

Comment:
Gotcha, it was the implicit dependency from `capnpc-ts` to `capnp-ts` (it imports the build artifacts with `../lib`, instead of directly referencing the source). Fixed it.